### ISSUE TITLE
Bug: Deno install error: @eslint/css is missing a prefix

### ISSUE THREAD

**Issue by MikeMcC399** [eslint/css#66]
### Environment

ESLint version:  N/A
@eslint/css version: `0.4.0 `
Node version:  `v22.14.0` LTS
Deno version:  `2.2.1`
Operating System: Ubuntu `24.04.2` LTS

### Which language are you using?

N/A

### What did you do?

```shell
npm install -g deno
cd $(mktemp -d)
deno add @eslint/css
```

### What did you expect to happen?

The [README > Installation](https://github.com/eslint/css/blob/main/README.md#installation) instructions should lead to a successful installation under Deno.

### What actually happened?

Error message:

```text
deno add @eslint/css
Created deno.json configuration file.
error: @eslint/css is missing a prefix. Did you mean `deno add jsr:@eslint/css`?
```

### Link to Minimal Reproducible Example

See above

### Participation

- [x] I am willing to submit a pull request for this issue.

### Additional comments

Probably the [README > Installation](https://github.com/eslint/css/blob/main/README.md#installation) is incorrect and the command should be:

```shell
deno add npm:@eslint/css
```

however this is also not successful. It results in:

```shell
$ deno add npm:@eslint/css
Add npm:@eslint/css@0.4.0
error: Error in @eslint/css@0.4.0 parsing version requirement for dependency "@types/css-tree": "file:./typings/css-tree"

Caused by:
    0: Invalid version requirement
    1: Unexpected character.
         file:./typings/css-tree
         ~
```

Installing an earlier release is successful:

```shell
deno add npm:@eslint/css@0.2.0
```

### Related

- #56 
- #58

**Comment by nzakas** [comment]
We do also publish to JSR, so not sure whether the advice should be to install from npm or JSR at this point. Probably JSR?

**Comment by MikeMcC399** [comment]
@nzakas 

> We do also publish to JSR, so not sure whether the advice should be to install from npm or JSR at this point. Probably JSR?

Probably you are right.

I notice though that https://eslint.org/docs/latest/use/getting-started does not list instructions for installing `eslint `and `@eslint/js` using Deno. The same for https://eslint.org/docs/latest/use/command-line-interface which does not cover how to run ESLint under Deno.

I have never used Deno before, so I wouldn't be the right person to be filling in these documentation gaps.

**Comment by MikeMcC399** [comment]
## Logs

with Deno `2.2.2` on Ubuntu `24.04.2` LTS
- `jsr` works
- `npm` fails

so it looks like `jsr` would be the better recommendation:

### jsr

```text
$ deno add jsr:@eslint/css
Created deno.json configuration file.
Add jsr:@eslint/css@0.4.0
```

### npm

```text
$ deno add npm:@eslint/css
Created deno.json configuration file.
Add npm:@eslint/css@0.4.0
error: Error in @eslint/css@0.4.0 parsing version requirement for dependency "@types/css-tree": "file:./typings/css-tree"

Caused by:
    0: Invalid version requirement
    1: Unexpected character.
         file:./typings/css-tree
         ~
```

**Comment by MikeMcC399** [comment]
@nzakas 

In that case I suggest to change the command in the README to `deno add jsr:@eslint/css` and mark it with "experimental status".

**Comment by nzakas** [comment]
Sounds good to me. Do you want to make that change?

**Comment by MikeMcC399** [comment]
@nzakas 

> Sounds good to me. Do you want to make that change?

- I have submitted https://github.com/eslint/css/pull/71 for this change.

**Comment by MikeMcC399** [comment]
- I just noticed that @xbinaryx has submitted #70 😕 so you now have two PRs to choose from.
